doc/contributing/bazel_notes.md

```diff
@@ -116,110 +117,144 @@ All device software, regardless of the device, is built with Bazel.
-  ```console
+  ```sh
   bazel test --test_tag_filters=verilator //sw/device/silicon_creator/...
   ```
 * Run a single ROM functional or E2E test on FPGA and see the output in real time:
-  ```console
+  ```sh
   bazel test \
     --define DISABLE_VERILATOR_BUILD=true \
     --test_tag_filters=cw310 \
     --test_output=streamed \
     //sw/device/silicon_creator/lib:boot_data_functest
   ```
   or, remove the define/filtering flags and just append the `<device>` name like:
-  ```console
+  ```sh
   bazel test --test_output=streamed //sw/device/silicon_creator/lib:boot_data_functest_fpga_cw310
   ```
   and similarly for Verilator:
-  ```console
+  ```sh
   bazel test --test_output=streamed //sw/device/silicon_creator/lib:boot_data_functest_sim_verilator
   ```
 
 ### Chip-Level Tests
+
 * Query for all chip-level tests for FPGA:
-  ```console
+  ```sh
   bazel query 'filter(".*_fpga_cw310", kind(".*test rule", //sw/device/tests/...))'
   ```
   and for Verilator:
-  ```console
+  ```sh
   bazel query 'filter(".*_sim_verilator", kind(".*test rule", //sw/device/tests/...))'
   ```
 * Run all chip-level tests on FPGA:
-  ```console
+  ```sh
   bazel test --define DISABLE_VERILATOR_BUILD=true --test_tag_filters=cw310 //sw/device/tests/...
   ```
   and for Verilator:
-  ```console
+  ```sh
   bazel test --test_tag_filters=verilator //sw/device/tests/...
   ```
 * Run a single chip-level test on FPGA and see the output in real time:
-  ```console
+  ```sh
   bazel test \
     --define DISABLE_VERILATOR_BUILD=true
     --test_tag_filters=cw310 \
     --test_output=streamed \
     //sw/device/tests:uart_smoketest
   ```
   or, remove the define/filtering flags and just append the `<device>` name like:
-  ```console
+  ```sh
   bazel test --test_output=streamed //sw/device/tests:uart_smoketest_fpga_cw310
   ```
   and similarly for Verilator:
-  ```console
+  ```sh
   bazel test --test_output=streamed //sw/device/tests:uart_smoketest_sim_verilator
   ```
 
 # Linting Software
 
 There are several Bazel rules that enable running quality checks and fixers on code.
+They can be listed by running the following:
+```sh
+bazel query //quality:all
+```
 The subsections below describe how to use them.
 All of the tools described below are run in CI on every pull request, so it is best to run them before committing code.
 
 ## Linting C/C++ Code
+
 The OpenTitan supported linter for C/C++ files is `clang-format`.
 It can be run with Bazel as shown below.
 
 Run the following to check if all C/C++ code as been formatted correctly:
-```console
+```sh
 bazel test //quality:clang_format_check --test_output=streamed
 ```
 and run the following to fix it, if it is not formatted correctly.
-```console
+```sh
 bazel run //quality:clang_format_fix
 ```
 
+## Linting Rust Code
+
+The OpenTitan supported linter for Rust files is `cargo fmt`.
+It can be run with Bazel as shown below.
+
+Run the following to check if all C/C++ code as been formatted correctly:
```

Comment:
```suggestion
## Formatting Rust Code

The OpenTitan project uses `rustfmt` to format Rust code.
It can be run with Bazel as shown below.

Run the following to check if all Rust code as been formatted correctly:
```

*(We use clippy as our linter, which is run a build time by default.)*

Reply:
Fixed.